src/main/java/uk/gov/hmcts/darts/retention/service/RetentionServiceImpl.java

```
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.darts.retention.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.darts.common.entity.CaseRetentionEntity;
+import uk.gov.hmcts.darts.common.repository.CaseRetentionRepository;
+import uk.gov.hmcts.darts.retention.mapper.RetentionMapper;
+import uk.gov.hmcts.darts.retentions.model.CaseRetention;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class RetentionServiceImpl implements RetentionService {
+    private final CaseRetentionRepository caseRetentionRepository;
+    private final RetentionMapper retentionMapper;
+
+    @Override
+    public List<CaseRetention> getCaseRetentions(Integer caseId) {
+        List<CaseRetentionEntity> caseRetentionEntities =
+            caseRetentionRepository.findByCaseId(caseId);
+
+        List<CaseRetention> caseRetentions = new ArrayList<>();
```

If no cases are found, do we want to return an error?